src/main/kotlin/no/nav/familie/ks/sak/kjerne/brev/begrunnelser/IBegrunnelse.kt

```diff
@@ -15,6 +15,10 @@ sealed interface IBegrunnelse {
 
     fun enumnavnTilString(): String
 
+    fun støtterFritekst(sanityBegrunnelser: List<SanityBegrunnelse>) =
+        sanityBegrunnelser.firstOrNull { it.apiNavn == this.sanityApiNavn }?.støtterFritekst ?: false ||
+            (this !== NasjonalEllerFellesBegrunnelse.REDUKSJON_SATSENDRING && this.begrunnelseType == BegrunnelseType.REDUKSJON)
```

Comment:
Logikk hentet fra frontend.

Reply:
Usikker på om vi trenger å sjekke at begrunnelsetypen er reduksjon, for det skjønner man egentlig ut fra reduksjon satsendring